### PR TITLE
Add `keyboardWillChangeFrame` and  `keyboardDidChangeFrame` event

### PR DESCRIPTION
**Pull request by magicismight**
Add `keyboardWillChangeFrame` and  `keyboardDidChangeFrame` event

### CODE REVIEW

File: KeyboardEvents.ios.js [hunk 1]
```diff
@@ -34,10 +34,26 @@ RCTDeviceEventEmitter.addListener(
...
+  RNKeyboardEventsManager.KeyboardWillChangeFrame,
+  (frames) => {
+    KeyboardEventEmitter.emit(RNKeyboardEventsManager.KeyboardWillChangeFrame, frames);
+  }
+);
+
+RCTDeviceEventEmitter.addListener(
+  RNKeyboardEventsManager.KeyboardWillChangeFrame,
+  (frames) => {
+      KeyboardEventEmitter.emit(RNKeyboardEventsManager.KeyboardDidChangeFrame, frames);
```
Comment (johanneslumpe) [inline KeyboardEvents.ios.js:47]: Could you fix this identation? :)

**Comment by johanneslumpe** [comment]
Could it be that you broke the xcode project file? I'm seeing path issues now which prevent me from compiling.

```
react-native-keyboardevents/RNKeyboardEvents/RNKeyboardEvents.m'
clang: error: no input files
```

That file doesn't even exist, nor does the directory.

**Comment by magicismight** [comment]
Perhaps…
sorry，I'm new to iOS development.
Maybe I had broken the Xcode project.

**Comment by magicismight** [comment]
Maybe it is better that you add this change to the project.
I will close this PR.
I`m mess with iOS development..sorry

**Pull request closed without merging**